src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -70,10 +70,17 @@ Regenerate the directory by running `flutter clean` or `flutter pub get` in the
 There are two ways to embed Flutter in your existing application.
 
 1. Use the CocoaPods dependency manager and installed Flutter SDK. (Recommended.)
-2. Create frameworks for the Flutter engine, your compiled Dart code, and all
+1. Create frameworks for the Flutter engine, your compiled Dart code, and all
    Flutter plugins. Manually embed the frameworks, and update your existing
    application's build settings in Xcode.
 
+{{site.alert.note}}
+Your app will not run on a simulator with Flutter's Release framework. You can run
```

Comment:
since Flutter does not yet support output x86 ahead-of-time (AOT) binaries for your Dart code.

Reply:
Also, maybe just release mode instead of release framework, since people who choose option A won't necessarily be exposed to "frameworks".